docs/docsite/rst/community/other_tools_and_programs.rst

```diff
@@ -1,24 +1,108 @@
+The Ansible community provides several useful tools for working with the Ansible project. This is a list of some of the most popular of these tools.
+
+*******
+Editors
+*******
+
+Atom
+====
+
+Is a open source and free GUI text editor by GitHub. It can keep track of git project
+changes, can commit from the GUI and see what branch you are on. You can customize the
+themes for different colors and install packages to help with different languages Language
+YAML. You can install it on Linux, OSX and Windows.
+
+* `Language YAML <https://github.com/atom/language-yaml>`_
+
+
+Pycharm
+=======
+
+Is a full IDE to develop Python software. It has all the needed parts to be able to write
+python scripts and complete software. Is a little overkill for writing roles/playbooks but, it can
+be a very useful tool if you write modules and submit code for Ansible.
+
+
+Sublime
+=======
+
+You can customize GUI with themes, install packages and do the same type of work. You can install it on
+Linux, OSX and Windows
+
+* `GitGutter <https://packagecontrol.io/packages/GitGutter>`_ - plug-in to show information about files in a git repository
+* `SideBarEnhancements <https://packagecontrol.io/packages/SideBarEnhancements>`_ - Provides enhancements to the operations on Sidebar of Files and Folders
+* `Sublime Linter <https://packagecontrol.io/packages/SublimeLinter>`_ - The code linting framework for Sublime Text 3
+* `Pretty YAML <https://packagecontrol.io/packages/Pretty%20YAML>`_ - Prettify YAML plugin for Sublime Text 2 & 3
+* `Yamllint <https://packagecontrol.io/packages/SublimeLinter-contrib-yamllint>`_ - Sublime wrapper around yamllint
+
+VS Code
+=======
+
+Also similar like Atom & sublime but, made by Microsoft. Its also open source and free.
+
+
+* `YAML Support by Red Hat <https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml>`_ - Provides YAML support via yaml-language-server with built-in Kubernetes and Kedge syntax support.
+* `Ansible Syntax Highlighting Extension <https://marketplace.visualstudio.com/items?itemName=haaaad.ansible>`_ - YAML & Jinja2 support
+* `Visual Studio Code extension for Ansible <https://marketplace.visualstudio.com/items?itemName=vscoss.vscode-ansible>`_ - Autocompletion, syntax highlighting
+
+vim
+====
+
+* `Ansible vim <https://github.com/pearofducks/ansible-vim>`_  - vim syntax plugin for Ansible 2.x, it supports YAML playbooks, Jinja2 templates, and Ansible's hosts files.
 
-- `Ansible Lint <https://github.com/willthames/ansible-lint>`_ is a widely used, highly configurable best-practices linter for Ansible playbooks.
 
+********************
+Validating Playbooks
+********************
+
+- `Ansible Lint <https://github.com/willthames/ansible-lint>`_ is a widely used, highly configurable best-practices linter for Ansible playbooks.
 - `Ansible Review <http://willthames.github.io/2016/06/28/announcing-ansible-review.html>`_ is an extension of Ansible Lint designed for code review.
```

Comment:
Probably good to change the URL to point to the repo which aligns with the new entries you've added https://github.com/willthames/ansible-review